modules/virt-about-static-and-dynamic-ssh-keys.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+:_content-type: REFERENCE
+[id="virt-about-static-and-dynamic-ssh-keys_{context}"]
+= About static and dynamic public keys
+
+You can add static or dynamic public SSH keys to virtual machines (VMs).
+
+[discrete]
+[id="static-public-key_{context}"]
+== Static public key
+
+A static public SSH key is added to the VM at startup. It is propagated by using a cloud-init configuration drive and does not affect cloud-init user data.
+
+All operating systems support static public keys.
```

Comment:
It is required that the guest is able to apply the configuration from the cloud-init data soruce. This is the case for the RHEL cloud image, but also many 3rd party cloud images. It is even possible to use cloud-init in windows, e.g. via https://cloudbase.it/cloudbase-init/ .